templates/students/update_profile.html

```diff
@@ -149,4 +152,33 @@ <h2>
         }, false)
     }())
 </script>
+
+<!-- Script for Image Validation and auto-submitting the form-->
+<script type="text/javascript">
+    function imageValidation() {
+        var fileInput = document.getElementById('profileImage');
+        var filePath = fileInput.value;
+        var allowedExtensions = /(\.jpg|\.jpeg|\.png|\.gif)$/i;
+        if (!allowedExtensions.exec(filePath)) {
+            alert('Please upload file having extensions .jpeg, .jpg, .png or .gif only.');
+            fileInput.value = '';
+        }
+        else{
+            document.getElementById("submit").click();
+        }
+    }
+</script>
+
+<!--Scripts for popover feature-->
+
+<script>
+    $(function () {
+        $('.dismissible-popover').popover({
+            sanitize : false,
+            trigger : 'focus',
+            html : true,
+            placement : 'bottom'
```

Comment:
Here would be the right place to add the template for popover. Also, as you are already mentioning the `html` attribute here, no need to mention the same `data-html` attribute above.